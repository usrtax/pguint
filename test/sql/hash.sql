-- reference
SELECT hashint4(55);
SELECT hashoid(55);

SELECT hashint1(55::int1);
SELECT hashu8(55::u8);
SELECT hashu16(55::u16);
SELECT hashu32(55::u32);
SELECT hashu64(55::u64);
