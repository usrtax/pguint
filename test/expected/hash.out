-- reference
SELECT hashint4(55);
  hashint4  
------------
 -305105437
(1 row)

SELECT hashoid(55);
  hashoid   
------------
 -305105437
(1 row)

SELECT hashint1(55::int1);
  hashint1  
------------
 -305105437
(1 row)

SELECT hashu8(55::u8);
 hashu8  
------------
 -305105437
(1 row)

SELECT hashu16(55::u16);
 hashu16  
------------
 -305105437
(1 row)

SELECT hashu32(55::u32);
 hashu32  
------------
 -305105437
(1 row)

SELECT hashu64(55::u64);
 hashu64  
------------
 -305105437
(1 row)

